controllers/ocimachine_controller.go

```diff
@@ -316,12 +317,13 @@ func (r *OCIMachineReconciler) recordErrorsOnFailedWorkRequest(ctx context.Conte
 				r.Recorder.Event(machine, corev1.EventTypeWarning, "ReconcileError", errors.Wrapf(err, "Failed to normal reconcile OCIMachine").Error())
 				return reconcile.Result{}, errors.Wrapf(err, "failed to normal reconcile OCI Machine %s/%s", machineScope.OCIMachine.Namespace, machineScope.OCIMachine.Name)
 			}
+			final_error = final_error + fmt.Sprintf("Workrequest %s failed.", *wrqst.Id)
 			for _, wr_err := range wr_errs.Items {
-				r.Recorder.Eventf(machine, corev1.EventTypeWarning, "ReconcileError", *wr_err.Message)
+				final_error = final_error + " " + *wr_err.Message
```

Comment:
String formatting the Go way is typically using `fmt.Sprintf` like `fmt.Sprintf("%s %s", final_error, *wr_err.Message)` or whatever. I haven't tested that.

Reply:
Also by removing `r.Recorder.Eventf` isn't functionality lost? Isn't this actually recording an event here?

OH! this is the expected change? So this is just going to build a long error of all the possible error messages in the WR?